airflow-core/src/airflow/jobs/triggerer_job_runner.py

```diff
@@ -311,8 +352,25 @@ def _handle_request(self, msg: ToTriggerSupervisor, log: FilteringBoundLogger) -
                 if factory := self.logger_cache.pop(id, None):
                     factory.upload_to_remote()
             return
```

Comment:
Does this `return` cause any issue?

Reply:
It does prettty much nothing, it's a return only that will save a boolean check down the line `if resp:` (resp is always None in this case).

Not needed and maybe confusing, removed.